templates/src/app.js

```diff
@@ -27,6 +29,15 @@ app.use("/auth", authRoutes);<% end %>
 
 const port = process.env.SERVER_PORT || 8080;
 
+<%if eq (index .Params `cacheStore`) "memcached" %>
+function touchMemcache() {
+  var memcached = cacheStore;
+  memcached.set('foo', 'bar', 10, function (err) {if (err) throw new Error(err); console.log("Set operation successful")});
```

Comment:
maybe just use the `cacheStore` variable instead of reassigning it? 
and can we separate the callback into its own line, maybe like 
```js
memcached.set('foo', 'bar', 10,
  function (err) {
    if (err) throw new Error(err); 
    console.log("Set operation successful");
  });
```

can we also include a comment like the db part telling them 
```js
// memcached example getting and setting key value
```

Reply:
Also maybe let's call this function inside `main` below?

Reply:
And the naming `cacheStore` implies that it will eventually provide both Memcached and Redis functionality but then this code implies that it's only for Memcached. Since at some point Redis compatibility will be added maybe it's worth restructuring a bit to make it more agnostic.

Reply:
Thanks for the feedback guys! I just updated the code.